src/Fixer/Casing/ClassReferenceNameCasingFixer.php

```diff
@@ -81,7 +81,8 @@ private function getClassReference(Tokens $tokens, NamespaceAnalysis $namespace)
         static $blockKinds;
 
         if (null === $notBeforeKinds) {
-            $notBeforeKinds = [
+            $notBeforeKinds = Token::getObjectOperatorKinds();
+            $notBeforeKinds += [
```

Comment:
are you sure you want to use union for non-assoc arrays?
https://3v4l.org/8sm9t

Reply:
I will take a look :+1: